start-here/js/script-main.js

```diff
@@ -6,101 +6,137 @@ const input = document.querySelector("#search-field");
 //Select button-search:
 const searchButton = document.querySelector(".search-button");
 
-//Add event listner to the search button
+// audio element and play image
+const pronunciationAudio = document.querySelector("audio");
+const playImage = document.getElementById("play-image");
+
+//Add event listener to the search button
 searchButton.addEventListener("click", handleSearchButtonClick);
 
 input.addEventListener("focus", handleSearchInputFocus);
 
 //Handle Search
-function handleSearchButtonClick() {
+async function handleSearchButtonClick() {
   if (input.value === "") {
-
     warningMessage.classList.remove("warning-message--hidden");
+    // add display: block to show a message
+    warningMessage.style.display = "block";
     input.classList.add("search-input--warning");
     return;
-    
   } else {
-    warningMessage.style.display = "none";
-    fetch(`https://api.dictionaryapi.dev/api/v2/entries/en/${input.value}`)
-      .then((response) => response.json())
-      .then((data) => {
-        if (data.title) {
-          document.getElementById(
-            "title-wrapper"
-          ).innerHTML = `<p>No definition found for <strong>${input.value}</strong>.</p>`;
+    const response = await fetch(
+      `https://api.dictionaryapi.dev/api/v2/entries/en/${input.value}`
+    );
+    // need to check for 404 error
+    if (response.ok) {
+      const data = await response.json();
+      const meanings = data[0].meanings;
+      const phonetics = data[0].phonetics[0];
+
+      document.querySelector(".title").innerText = input.value;
+
+      if (phonetics.text) {
+        // use specific selector
+        const h2 = document.querySelector(".phonetic-subtitle");
+        h2.innerText = phonetics.text;
+        const audioWrapper = document.querySelector(".audio-wrapper");
+        // conditional play audio display
+        if (phonetics.audio.length) {
+          audioWrapper.style.display = "block";
+          const audio = document.querySelector("audio");
+          audio.src = phonetics.audio;
```

Comment:
The audio button is gone after searching for a word:
![image](https://github.com/user-attachments/assets/d9174ac7-fead-484e-99bc-6c50903e0f0f)

Reply:
I've done it for words that don't have audio so the user will not be confused. If you search for a star, it will have button appear again.